man/ZydisInfo.1.ronn

```diff
@@ -0,0 +1,30 @@
+ZydisInfo(1) -- disassemble in your terminal
+============================================
+
+## SYNOPSIS
+
+`ZydisInfo -[real|16|32|64] <hexbytes>`
```

Comment:
Current syntax is:
```
ZydisInfo <machine_mode> [stack_width] <hexbytes>
Machine mode:      -real|-16|-32|-64
Stack width:       -16|-32|-64
```

Reply:
Do these options also apply to ZydisDisasm?

Reply:
For ZydisDisasm I never implemented the stack-width option if I remember correctly.

Might be a nice addition tho. Will keep that in mind for future improvement.